src/Overlay/Window/framedata.h

```diff
@@ -26,14 +26,34 @@ struct PlayersInteractionState
 	int p2GapDisplay = -1;
 };
 
+struct CharDataExtended
```

Comment:
rename it to something that doesn't make it sound like a natural memory structure, maybe something like playerData or whatever you feel its appropriate, just concerned about it creating confusion with CharData in the future in case we end up having to extend it for subclassing or something